Esri/arcgis-maps-sdk-unreal-engine-samples
# Fix UE 5.3 Routing Issue and Weather Sample Crashing

**Sample**

Weather Sample & Routing Sample & Material By Attribute

**Summary**

Routing Sample was keeping users from being able to open the project in newer versions of Unreal Engine, specifically 5.3
Weather Sample was crashing due to memory issues. 
Material By Attribute Building Name Renderer was missing the Windows texture to render the material on the 2 buildings

**Tests**

- Right-click on the .Uproject and click "change unreal engine version"
- Change it to UE 5.3
- Open the .sln file and build it, it should succeed.
- Open UE 5.3 and test samples

**ArcGIS Maps SDK Version**

UE 5.3 w/ Maps SDK 1.4
